Automattic/wp-calypso
# Virtual themes: Preview custom content

Fixes https://github.com/Automattic/wp-calypso/issues/75493
Requires D107928-code

## Proposed Changes

Generate previews of virtual designs with custom content based on the `preview_data` info returned by the `starter-designs` endpoint (see D107928-code).

This way, we can preview different content for virtual designs that has the same dynamic content internally.

Before | After
--- | ---
<img width="1022" alt="Screenshot 2023-04-17 at 11 59 01" src="https://user-images.githubusercontent.com/1233880/232452284-b6ba09bc-9e79-49b2-8ae6-05b32553fb2f.png"> | <img width="1006" alt="Screenshot 2023-04-17 at 12 00 01" src="https://user-images.githubusercontent.com/1233880/232452257-5633f791-4b9b-4cdc-bd62-c1a1275b540e.png">


## Testing Instructions

- Use the Calypso live link below.
- Go to `/setup/site-setup/designSetup?siteSlug=<SITE_DOMAIN>`.
- Make sure that all Link in Bio virtual designs no longer display the same content.
